src/tooling/debugging/index.md

```diff
@@ -4,12 +4,36 @@ Debugging Rust applications is also possible using different tools that will be
 
 Refer to the table below to see which chip is supported in every debugging method:
 
-|              | **probe-rs** | **OpenOCD** | **VS Code** |
-| :----------: | :----------: | :---------: | :---------: |
-|  **ESP32**   |      ❌       |      ✅      |      ✅      |
-| **ESP32-C2** |      ✅       |      ✅      |      ✅      |
-| **ESP32-C3** |      ✅       |      ✅      |      ✅      |
-| **ESP32-C6** |      ✅       |      ✅      |      ✅      |
-| **ESP32-H2** |      ✅       |      ✅      |      ✅      |
-| **ESP32-S2** |      ❌       |      ✅      |      ✅      |
-| **ESP32-S3** |      ❌       |      ✅      |      ✅      |
+|              | **probe-rs** | **OpenOCD** |
+| :----------: | :----------: | :---------: |
+|  **ESP32**   |      ❌       |      ✅      |
```

Comment:
Since Xtensa support is actively being worked on, maybe we can instead use ⏳ to indicate such, which is consistent with other chapters in the book.

Reply:
Good idea!